RecoLocalCalo/HGCalRecAlgos/src/HGCal3DClustering.cc

```diff
@@ -102,9 +102,13 @@ std::vector<reco::HGCalMultiCluster> HGCal3DClustering::makeClusters(const reco:
 
       }
       if( temp.size() > minClusters ) {
+	math::XYZPoint position = clusterTools->getMultiClusterPosition(temp);
+	if (abs(position.z()) <= 0.) continue;
```

Comment:
hi @clelange - please use std::abs - but it would seem that in a future PR, the interface of getMultiClusterPosition should be changed to allow you to catch this error condition via a bool comparison.